test/react16_suggestors/react_style_maps_updater_test.dart

```diff
@@ -0,0 +1,569 @@
+      testSuggestor(
+        expectedPatchCount: 0,
+        input: '''
+          $classSetupBoilerPlate
+            main() {
+              Foo()
+              ${getCheckboxComment(keysOfModdedValues: ['width'])}
+              ..style = {
+                'width': '40px',
+              };
+            }
+        ''',
+      );
+    });
+
+    test('does not add a second validate comment when checked', () {
+      testSuggestor(
+        expectedPatchCount: 0,
+        input: '''
+          $classSetupBoilerPlate
+            main() {
+              Foo()
+              ${getCheckboxComment(keysOfModdedValues: [
+          'width'
+        ], checked: true)}
+              ..style = {
+                'width': '40px',
+              };
+            }
+        ''',
+      );
+    });
+
+    test('does not override comments', () {
+      testSuggestor(
+        expectedPatchCount: 4,
+        input: '''
+            $classSetupBoilerPlate
+            main() {
+              Foo()
+              ..style = {
+                'width': '40',
+                // Test Comment
+                'height': '40%',
+                'fontSize': '12',
+                'margin': '25',
+              };
+            }
+          ''',
+        expectedOutput: '''
+            $classSetupBoilerPlate
+            main() {
+              Foo()
+              ${getCheckboxComment(keysOfModdedValues: [
+          'width',
+          'fontSize',
+          'margin'
+        ])}
+              ..style = {
+                'width': 40,
+                // Test Comment
+                'height': '40%',
+                'fontSize': 12,
+                'margin': 25,
+              };
+            }
+          ''',
+      );
+    });
+
+    test('does not run on setProperty', () {
+      testSuggestor(
+        expectedPatchCount: 0,
+        input: '''
+            $classSetupBoilerPlate
+            main() {
+              DivElement()..style.setProperty('width', '400');
+            }
+          ''',
+        expectedOutput: '''
+            $classSetupBoilerPlate
+            main() {
+              DivElement()..style.setProperty('width', '400');
+            }
+          ''',
+      );
+    });
+
+    test('adds a comment for custom props', () {
+      testSuggestor(
+        expectedPatchCount: 1,
+        input: '''
+            $classSetupBoilerPlate
+            main() {
+              Foo()
+              ..datePickerProps = {
+                style: {
+                  'width': '40',
+                }
+              };
+            }
+          ''',
+        expectedOutput: '''
+            $classSetupBoilerPlate
+            main() {
+              Foo()
+              ${manualVariableCheckComment()}
+              ..datePickerProps = {
+                style: {
+                  'width': '40',
+                }
+              };
+            }
+          ''',
+      );
+    });
+  });
+}
+
+String getCheckboxComment({
+  bool checked: false,
```

Comment:
default args in dart2 prefer using `=` instead of `:`

Reply:
Nice, I didn't know that. I do like that better.